<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>271</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Events</string>
     </attribute>
     <widget class="QTableWidget" name="t_events">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>521</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <column>
       <property name="text">
        <string>Event</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Exp</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="b_event_del">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>540</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>331</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>New Event</string>
      </property>
      <widget class="QTextEdit" name="e_event_desc">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>311</width>
         <height>81</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="l_event_desc">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="s_exp">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>281</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="l_event_exp">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Experience for finishing</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="e_event_title">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>311</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="l_event_title">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>59</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
      <widget class="QPushButton" name="b_add_event">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Event</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="l_event_disp_desc">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>330</y>
        <width>331</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Skills</string>
     </attribute>
     <widget class="QTreeView" name="treeView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>551</width>
        <height>551</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Achievments</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>571</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>251</width>
        <height>571</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Status</string>
     </attribute>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>551</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
